user/session.go

```diff
@@ -112,6 +180,21 @@ type Monitor struct {
 	TriggerLimits []float64 `json:"trigger_limits" msg:"trigger_limits"`
 }
 
+type Endpoint struct {
+	Path    string           `json:"path,omitempty" msg:"path"`
+	Methods []EndpointMethod `json:"methods,omitempty" msg:"methods"`
+}
+
+type EndpointMethod struct {
+	Name  string                  `json:"name,omitempty" msg:"name,omitempty"`
+	Limit EndpointMethodRateLimit `json:"limit,omitempty" msg:"limit,omitempty"`
+}
+
+type EndpointMethodRateLimit struct {
```

Comment:
Reuse the existing type of apidef?

Reply:
the existing apidef definition has other parameters in it, including method, path etc.